apps/web/src/server/modules/auth/auth.service.ts

```diff
@@ -98,7 +115,7 @@ export const emailVerifyOtp = async ({
     // Valid token, delete record to prevent reuse
     return db.verificationToken.delete({
       where: {
```

Comment:
@karrui how are unused verification tokens from previous attempts cleaned up?

Seems like they don't get auto-deleted since the identifier now includes the nonce.